src/classes/Defaults/MergeTag/Comment/CommentAuthorIP.php

```diff
@@ -45,7 +52,7 @@ public function __construct( $params = [] ) {
 				'description' => '127.0.0.1',
 				'example'     => true,
 				'resolver'    => function( $trigger ) {
-					return $trigger->comment->comment_author_IP;
+					return $trigger->{ $this->comment_type }->comment_author_IP;
```

Comment:
Shouldn't this be `property_name`? You are missing the assignment as well

Reply:
@Kubitomakita uh, I've missed that